librz/asm/arch/luac/v53/analysis_53.c

```diff
@@ -37,7 +37,7 @@ int lua53_anal_op(RzAnalysis *anal, RzAnalysisOp *op, ut64 addr, const ut8 *data
 			op->size = 8;
 		}
 		break;
-	case OP_LOADBOOL: /*  A B C   R(A) := (Bool)B; if (C) pc++                    */
+	case OP_LOADBOOL: /*  A B C   R(A) := (RzILBool)B; if (C) pc++                    */
```

Comment:
I think these substitutions in the Lua code are accidental and shouldn't happen, right?